jest.config.js

```diff
@@ -0,0 +1,7 @@
+module.exports = {
+  roots: ['<rootDir>/src'],
+  testMatch: ['**/__tests__/**/*.+(ts|js)', '**/?(*.)+(spec|test).+(ts|js)'],
+  transform: {
+    '^.+\\.(ts)$': 'ts-jest',
+  },
+};
```

Comment:
All these PRs have a lot of bootstrapping: They all seems reasonable, but if I were to come back to it, I'd be hard-pressed to to say what the motivation for any particular choice is. If there are any corners that aren't just obviously the right thing to do, commenting in the PR or the file iteself could helpful. (something like: "Configuration from the tutorial at this url ..." / "Configuration based on this stackoverflow ..." / "Configuration generated by this init command...")

Use your judgement.

Reply:
Filed #7